android/src/main/jni/CMakeLists.txt

```diff
@@ -26,12 +26,21 @@ target_include_directories(
 )
 
 find_package(ReactAndroid REQUIRED CONFIG)
-
+if (ReactAndroid_VERSION_MINOR GREATER_EQUAL 76)
 target_link_libraries(
```

Comment:
Let's add two spaces of indent here

Reply:
Done in [be14680](https://github.com/software-mansion/react-native-gesture-handler/pull/3109/commits/be146807078a1126e5956bd481f213495c519899)